blog-mysql-seed/mysql/mysql.star

```diff
@@ -0,0 +1,52 @@
+MYSQL_IMAGE = "mysql:8.0.32"
+
+def create_database(plan, database_name, database_user, database_password, seed_script_artifacts = []):
+    files = {}
+    for index, artifact in enumerate(seed_script_artifacts):
+        files["/docker-entrypoint-initdb.d/{}.sql".format(index)] = artifact
```

Comment:
Can we store this into a constant with a named explaining that this is? I am personally not sure what this does

Reply:
Also, is that expected to have files like `1.sql` `2.sql`, etc? Can we add a prefix that is a little bit descriptive maybe?

Reply:
they are sorted, so prepending the number guarantees that the order we execute it. Will add a comment to clarify it